jmx-scraper/src/test/java/io/opentelemetry/contrib/jmxscraper/config/PropertiesCustomizerTest.java

```diff
@@ -17,16 +17,20 @@
 
 class PropertiesCustomizerTest {
 
+  private static final String DUMMY_URL = "service:jmx:rmi:///jndi/rmi://host:999/jmxrmi";
```

Comment:
[for reviewer] we now have to test with an URL that can be parsed because the parsing is triggered, while it used to not be parsed previously.